services/payments/src/simcore_service_payments/core/settings.py

```diff
@@ -87,3 +92,14 @@ class ApplicationSettings(_BaseApplicationSettings):
         description="Password for HTTP Basic Auth. Required if started as a web app.",
         min_length=10,
     )
+
+    PAYMENTS_ACCESS_TOKEN_SECRET_KEY: SecretStr = Field(
+        ...,
+        description="To generate a random password with openssl in hex format with 32 bytes, run `openssl rand -hex 32`",
+        min_length=63,
+    )
+    PAYMENTS_ACCESS_TOKEN_EXPIRE_MINUTES: PositiveFloat = Field(default=30)
```

Comment:
@GitHK no it does not restrict the user to 30 mins for the payment. 
That API is only one part of the payment workflow.
This is an industry standard: OAuth2 with Password and Bearer (w/ JWT tokens)
This is the expiration of JWT tokens and the client can implement a mechanism (denoted "password flow") to renovate the cookie automatically when it expires ... so theoretically, the client has no limited access ... only the bad guys

Reply:
> minor: also for the ENV that are times, I now tend to use the timedelta type. This allows to put seconds or real times which are handy. It also allows to remove minutes at the end and use human readable times. see https://docs.pydantic.dev/latest/usage/types/datetime/

@sanderegg I actually tried your suggestion but since I wanted minutes I found more simple to set a possitive float. If I would user `teimdelta` I had [two options](https://docs.pydantic.dev/1.10/usage/types/#datetime-types):
1. with `int` or `float` in seconds  (too tempting to put small values)
2. with `str` as `+00:MM` and other (too complicated)

Reply:
@pcrespov my question about being an industry standard was bout the 30 minutes.

Reply:
I saw it in one tutorial. I wil double check